libs/components/packages/src/schematics/rules/convert-definition-list-to-description-list/convert-definition-list-to-description-list.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+import { firstValueFrom } from 'rxjs';
+
+import { createTestApp } from '../../testing/scaffold';
+
+import { convertDefinitionListToDescriptionList } from './convert-definition-list-to-description-list';
+
+describe('Convert Definition List to Description List', () => {
```

Comment:
Can we have a test to show what happens with the `labelWidth` input? https://github.com/blackbaud/skyux/blob/main/libs/components/layout/src/lib/modules/definition-list/definition-list.component.ts#L24

Reply:
There is no corollary for `labelWidth`, I'm intentionally dropping it, and I've added a warning message in the log.